src/sentry/integrations/vsts/urls.py

```diff
@@ -1,7 +1,8 @@
 from __future__ import absolute_import, print_function
 
-from django.conf.urls import patterns  # , url
-
+from django.conf.urls import patterns, url
+from .webhooks import WorkItemWebhook
 urlpatterns = patterns(
     '',
+    url(r'^webhook/$', WorkItemWebhook.as_view(), name='sentry-extensions-vsts-issue-updated'),
```

Comment:
We should `name` this something more generic like `sentry-extensions-vsts-webhook` to be consistent with the class and URL. We might want to use it for other events in the future.

Reply:
Pretty sure I was just following @macqueen example for jira there https://github.com/getsentry/sentry/blob/master/src/sentry/integrations/jira/urls.py#L17:L18 

We can change it to something else. I honestly don't have a good opinion here.

Reply:
That URL is `/extensions/jira/issue-updated`, though. Maybe we should rename this URL to match that specificity. I'd personally really love these to be consistent across all integrations that share this type of functionality.